.github/workflows/build.yml

```diff
@@ -0,0 +1,121 @@
+name: build
```

Comment:
Could you please add the "copyright message with apache license to this file, and also a high level note on when this workflow will be triggered"

Reply:
Does that need to be added to the CI workflow? Just curious cause I didn't see it in the `.travis.yml` file either. Happy to add if you think it's necessary though.

Reply:
Currently a miss on the .travis.yml. Shows up in fossa checks. As this file is going to get used as a template for other repos, would be good to add it here.

Reply:
Adding in next commit 👍